tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/WriteBuildOutputProps.cs

```diff
@@ -55,6 +60,14 @@ public override bool Execute()
                 .OrderBy(id => id.Id)
                 .ToArray();
 
+            var additionalAssets = (AdditionalAssetPaths ?? new string[0])
+                .Where(Directory.Exists)
+                .Where(path => Directory.GetDirectories(path).Count() > 0)
+                .Select(path => new {
+                    Name = new DirectoryInfo(path).Name + "Version",
```

Comment:
Switch to consistently use `Directory` methods?

Reply:
There's not a Directory method to get the name of the dir.